packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/wifi/config/config.controller.js

```diff
@@ -243,4 +246,55 @@ export default /* @ngInject */ function XdslModemWifiConfigCtrl(
         },
       );
   };
+
+  self.showQRcode = function showQRcode(row) {
+    const title = $translate.instant('xdsl_modem_wifi_config_modal_title');
+    // Retrieve QR code from APIv6
+    $http
+      .get(
+        `apiv6/xdsl/${$stateParams.serviceName}/modem/wifi/${row.wifiName}/qrCode`,
+      )
+      .then((response) => {
+        // Display modal with the QR code retrieved from APIv6
+        const qrcode = response.data;
+        $uibModal.open({
```

Comment:
Good practice is not to use $uibModal which is an old library but in fact a component with template / controller / routing with: 
    views: {
      modal: {
        component: 'xxxxxx',
      },
    },

Reply:
Change done.